lib/Sema/CheckedCAlias.cpp

```diff
@@ -390,7 +389,7 @@ class UpdateDependences : public RecursiveASTVisitor<UpdateDependences> {
   UpdateDependences(VarDecl *BoundsDecl,
                     Sema::BoundsDependencyTracker::DependentMap &Map,
                     bool Add) :
-    BoundsDecl(BoundsDecl), Map(Map), Add(Add) {}
+    Add(Add), BoundsDecl(BoundsDecl), Map(Map) {}
```

Comment:
Why do we need to change the initialization order here?

Reply:
There is a gcc compiler warning about fields not being initialized in the same order that they were declared.   The idea is to make it easier for programmers to read code that does initialization and to notice if initializers have been forgotten.